src/settings.js

```diff
@@ -0,0 +1,8 @@
+export const SELECT_ICON = 'selectIcon';
+
+export default new Map(Object.entries({
```

Comment:
Can we work around `Object.entries` for the moment? I know that support [looks good](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) but it's not that old yet in some cases (Safari, Edge) and I wouldn't rely on somebody having installed a polyfill. 
We could draw the line at ES7 stuff.

Reply:
I made this code a bit ruder but it will have a better crossbrowser support